src/platform/testing/e2e/cypress/support/commands/keyboardNavigation.js

```diff
@@ -17,7 +17,8 @@ const removeLeadingHash = name => name.replace(leadingHashRegex, '');
 // Use chooseSelectOptionUsingValue or chooseSelectOptionByTyping instead.
 Cypress.Commands.add('findOption', value => {
   cy.get(':focus').then($el => {
-    if ($el.val() !== value) {
+    // Check that an input is focused
+    if ($el[0].tagName === 'INPUT' && $el.val() !== value) {
```

Comment:
Added because I've seen some keyboard only e2e tests timing out after 40 minutes. When I tested locally, the focus can get stuck on the header inside the `va-radio` if the code is using `waitForRenderThenFocus`